app/models/manageiq/providers/kubevirt/infra_manager/vm/operations/power.rb

```diff
@@ -14,4 +14,54 @@ def raw_stop
       vm.stop
     end
   end
+
+  def raw_create_snapshot(name, desc = nil, memory = false)
```

Comment:
Recommend calling this snapshot_name instead of name, so that you can use `name` as the vm name below.

Reply:
Hi @agrare, please review the changes and provide your suggestions